### ISSUE TITLE
Backend language disappeared

### ISSUE THREAD

**Issue by GuyT07** [polylang/polylang#580]
## Prerequisites

- [ ] This is not a usage question (Those should be directed to the [community supported forum](https://wordpress.org/support/plugin/polylang), unless this is a question about Polylang Pro in which case you should [use the helpdesk](https://polylang.pro/support/)).
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest `master` branch of Polylang and the latest WordPress version.
- [ ] This bug happens with only Polylang plugin active
- [ ] This bug happens with a default WordPress theme active
- [ ] I can reproduce this bug consistently

## Steps to reproduce the issue

Unknown.

## Expected behavior and actual behavior

In the backend the language flag is missing in the 'Pages' overview. When navigating to Languages --> languages the locale and flag are not saved. 
![Screenshot 2020-09-30 at 08 05 45](https://user-images.githubusercontent.com/8415603/94648875-c1ffb000-02f3-11eb-8aa8-5763bb0ee1f0.png)

![Screenshot 2020-09-30 at 08 06 41](https://user-images.githubusercontent.com/8415603/94648917-e22f6f00-02f3-11eb-94e4-6066cc32d2cb.png)

![Screenshot 2020-09-30 at 08 07 01](https://user-images.githubusercontent.com/8415603/94648936-ed829a80-02f3-11eb-9f05-4eb2c45c33d9.png)

I already disabled all plugins to see if it was a combination of plugins that caused the behaviour, but it seems to be more like a database issue. For your notice: it is only in the backend. When navigating to a specific page and click edit I am able to edit the page. 

Does anybody knows which specific rows in which table I have to check to solve this problem? The customer does not want to go back to a previous backup.

**Comment by GuyT07** [comment]
Update: the full name and language are updating, locale and flag are not.

**Comment by manooweb** [comment]
Hello,

🤔 Very strange it doesn't work for only one language (English)

Have you try to edit the english language yet ?
I see that 93 publications are in English. When you edit one of them do you see the language metabox with the right informations about the English - dropdown correctly selected on English and flag ?

![image](https://user-images.githubusercontent.com/1003778/94662832-3a24a080-0309-11eb-88f0-adb24ecce9de.png)

To answer to your technical question languages are taxonomies name 'language' and thus stored in term_taxonomy WordPress table and their properties stored in their description column as serialized datas.

For example for US English you should have
```
a:3:{s:6:"locale";s:5:"en_US";s:3:"rtl";i:0;s:9:"flag_code";s:2:"us";}
```
To understand, you can compare with the other languages which are right

**Comment by GuyT07** [comment]
Hi, 

Thanks for your answer. I have tried to edit the language and the flag and locale are not updating. The full name and code are updating.

The dropdown does not show the flag either.

![Screenshot 2020-09-30 at 11 03 49](https://user-images.githubusercontent.com/8415603/94665582-a05ef280-030c-11eb-9dfd-018956cbdb4e.png)

I have tried to replace the row in the database, but the flag is still not showing up. Any other ideas are of course welcome :)

**Comment by GuyT07** [comment]
Update: fixed.

Comparing the dev db with the test db I saw a line was missing. 

I inserted `insert into wp_term_taxonomy values (4, 4, 'term_language', '', 0, 5);` and `insert into wp_terms values(4, 'English', 'pll_en', 0);`

<img width="626" alt="Screenshot 2020-09-30 at 11 17 43" src="https://user-images.githubusercontent.com/8415603/94667043-93430300-030e-11eb-8432-03dabb47f5ea.png">

Probably due to a migration or something.
